sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClient.java

```diff
@@ -148,14 +152,13 @@ public TextAnalyticsServiceVersion getServiceVersion() {
      * specified.
      *
      * @return A {@link Mono} containing a {@link Response} whose {@link Response#getValue() value} has the
-     * {@link DetectLanguageResult detected language} of the text.
+     * {@link DetectedLanguage detected language} of the text.
      *
      * @throws NullPointerException if {@code text} is {@code null}.
-     * @throws com.azure.ai.textanalytics.models.TextAnalyticsException if the response returned with
-     * an {@link com.azure.ai.textanalytics.models.TextAnalyticsError error}.
+     * @throws TextAnalyticsException if the response returned with an {@link TextAnalyticsError error}.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
```

Comment:
ReturnType.COLLECTION

Reply:
Why? Since we are not returning a collection of T. And it is Mono<T>, I believe SINGLE should be the correct one to use. Correct me if I am wrong.

Reply:
Yeah, this should be a single, I think I commented at the wrong line, ignore this one.